README.md

```diff
@@ -33,12 +33,11 @@ nile init
 ...
 ✨  Cairo successfully installed!
 ...
-✅ Dependencies successfully installed
 🗄  Creating project directory tree
 ⛵️ Nile project ready! Try running:
 ```
 
-This command creates the project directory structure and installs `cairo-lang`, `starknet-devnet`, `pytest`, and `pytest-asyncio` for you. The template includes a makefile to build the project (`make build`) and run tests (`make test`).
+This command provides a template with a makefile to build the project (`make build`) and run tests (`make test`).
```

Comment:
actually there's not a makefile anymore

Reply:
i would mention what the command does tho. it creates the project structure, sample files, etc. maybe we should also create an .env file? that'd be a different issue tho

Reply:
> i would mention what the command does tho

On it

> maybe we should also create an .env file

I think that's a great idea! #249